app/models/concerns/form526_claim_fast_tracking_concern.rb

```diff
@@ -101,21 +105,22 @@ def prepare_for_evss!
       Rails.logger.error "Contention Classification failed #{e.message}.", backtrace: e.backtrace
     end
 
-    measure_ep_merge_stats if disabilities.count == 1 && increase_or_new? && classification_updated
+    prepare_for_ep_merge! if disabilities.count == 1 && increase_only? && classification_updated
 
     return if pending_eps? || disabilities_not_service_connected?
 
     save_metadata(forward_to_mas_all_claims: true)
   end
 
-  def measure_ep_merge_stats
+  def prepare_for_ep_merge!
     pending_eps = open_claims.select { |claim| EP_MERGE_BASE_CODES.include?(claim['base_end_product_code']) }
     StatsD.distribution("#{EP_MERGE_STATSD_KEY_PREFIX}.pending_ep_count", pending_eps.count)
-    if pending_eps.count == 1
-      date = Date.strptime(pending_eps.first['date'], '%m/%d/%Y')
-      days_ago = (Time.zone.today - date).round
-      StatsD.distribution("#{EP_MERGE_STATSD_KEY_PREFIX}.pending_ep_age", days_ago)
-    end
+    return unless pending_eps.count == 1
+
+    date = Date.strptime(pending_eps.first['date'], '%m/%d/%Y')
+    days_ago = (Time.zone.today - date).round
+    StatsD.distribution("#{EP_MERGE_STATSD_KEY_PREFIX}.pending_ep_age", days_ago)
+    save_metadata(ep_merge_pending_claim_id: pending_eps.first['id'])
```

Comment:
This feature reuses the `save_metadata` method from RRD (Rapid Ready for Decision) which allows earlier parts of the 526 submission process to unobtrusively use the F526Submission's json blob to leave data for later parts.

In this case, EP400 merge eligibility is calculated right before establishment in EVSS, and if eligible the actual merge request is performed after establishment, since it requires the submitted claim ID.